DX-1322 Postel's Law (#790)
SwedbankPay/developer.swedbankpay.com: introduction.md

You can inspect [Swedbank Pay's TLS and cipher suite][ssllabs] support at
SSL Labs. Support for HTTP/2 in our APIs is being investigated.

> {% include alert.html type="informative" icon="info" body= "We encourage you to

Reviewer: I don't think this needs to be an alert, since everything said in the `introduction` is as important as this. Give it its own title, such as `Postel's Robustness Principle`. That's also going to fix the unrendered link (Markdown doesn't work in the `alert`'s `body`).

Reviewer: Please remove the `include alert.html`, make this a proper section and give it a header as suggested. :)

Author: Huh? I pulled it to VS code and did the changes there. I thought I commited the changes and pushed them. I'll do it again 😄

keep [Postel's robustness principle][robustness-principle] in mind. Build your
integration in a way that is resilient to change, wherever it may come. Don't
confine yourself to the limits of our current documentation examples. A `string`

> looking like a `guid` must still be parsed as a `string`, not as a guid, as it

Reviewer: ```suggestion
looking like a `guid` must still be handled and stored like a `string`, not as a guid, as it
```

could be a `URI` in the future. The day our `transactionNumber` ticks past
1,000,000, make sure your integration can handle number 1,000,001. If some

> `properties`, `operations` or `headers` can't be understood, you must be able to

Reviewer: ```suggestion
`fields`, `operations` or `headers` can't be understood, you must be able to
```

ignore them. We have built our requests in a way which allows the `payeeInfo`

> node to be placed before `metadata`, or vice versa if you want. We don't expect

Reviewer: ```suggestion
field to be placed before `metadata`, or vice versa if you want. We don't expect
```
